### ISSUE TITLE
Quest: Delivery Device

### ISSUE THREAD

**Issue by KraitX** [Valhalla-Gaming/Tracker#5228]
http://www.wowhead.com/quest=14339/delivery-device

Only 2 dying kodo's there should be at least 16 or more

http://www.wowhead.com/item=49195/hollow-kodo-horn

**Comment by Tregeta** [comment]
Incorrect, the locations on WoWhead are just where they have been killed at. This could mean only one thing, they have been kited to this position from their spawn point and killed..

Retail shows there are at least 7 spawned near here, which our database has collected this data.

**Comment by KraitX** [comment]
I See so if you go to wowhead http://www.wowhead.com/npc=4701/dying-kodo#comments
and it says 

> This NPC can be found in Desolace (28).

This doesnt mean then that there should be 28?

Anycase, I did find 3, you need 3 for the quest so i will close this issue.

**Comment by Tregeta** [comment]
28 are the recorded sightings through their parsing system, you can fly over an area and leave it and then come back to the same area and it will record them again if they have moved even a fraction of an inch.

Plan to look further into this to be certain there are identical amounts
